src/test/scala/is/hail/io/ExportVcfSuite.scala

```diff
@@ -101,16 +101,6 @@ class ExportVcfSuite extends SparkSuite {
     assert(hc.importVCF(out2).same(vds))
   }
 
-  @Test def testPPs() {
-    val vds = hc.importVCF("src/test/resources/sample.PPs.vcf", ppAsPL = true)
-    val out = tmpDir.createTempFile("exportPPs", ".vcf")
-    vds.exportVCF(out, exportPP = true)
```

Comment:
Since you didn't delete the `export_pp` option from `export_vcf`, I don't think you should delete this test. Or you could remove the option from export_vcf.